{{cookiecutter.project_name}}/{{cookiecutter.project_name}}/settings.py

```diff
@@ -60,7 +60,8 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
             'DATABASE_PASSWORD', default=default('{{cookiecutter.project_name}}')
         ),
         'HOST': env.str('DATABASE_HOST', default='localhost'),
-        'PORT': env.str('DATABASE_PORT', default='')
+        'PORT': env.str('DATABASE_PORT', default=''),
+        'OPTIONS': {'sslmode': env.str('DJANGO_DATABASE_OPTIONS_SSLMODE', default='prefer')}
```

Comment:
With #44 we have removed all `DJANGO_` prefix. So we should do it here as well.

Maybe we can make this even more flexible as environ also supports dict like `dict (BAR=key=val,foo=bar)`

This way we could call this option `DATABASE_OPTIONS`.

See https://github.com/joke2k/django-environ